src/main/java/com/parasoft/findings/jenkins/util/FilePathUtil.java

```diff
@@ -49,9 +49,9 @@ public static boolean isAbsolute(FilePath file)
         boolean result = false;
         try {
             result = file.act(new IsAbsoluteFileCallable());
-        } catch (IOException e) {
+        } catch (IOException e) { // parasoft-suppress OWASP2021.A9.LGE "This is intentionally designed to ensure exceptions during file processing don't cause the build to fail."
```

Comment:
It seems that `isAbsolute` method is only used in test. Maybe we can change the comment to: the exception is only used in tests and not rethrown because it's not critical.